makefile

```diff
@@ -27,6 +28,9 @@ endif
 LDFLAGS += $(LIBD_LIB) -lphobos2 $(LLVM_LIB)
 
 ifeq ($(PLATFORM),Linux)
+	ifeq ($(LD_GOLD),ld.gold)
+		LDFLAGS += -Wl,-fuse-ld=gold
```

Comment:
https://gcc.gnu.org/onlinedocs/gcc/Link-Options.html

This seems to say that -Wl is not necessary.